### ISSUE TITLE
[data-export] Prevent AggregateResult Column Being Clickable

### ISSUE THREAD

**Issue by Garywoo** [tprouvot/Salesforce-Inspector-reloaded#663]
**Is your feature request related to a problem? Please describe.**
When making a SOQL query including a `GROUP BY` statement, you'll get a column of `AggregateResult` links. The popup menu that is displayed when you click this link has non functional links in it.

![image](https://github.com/user-attachments/assets/4e972e9c-5241-460f-a9eb-c83e8b66c89e)

**Describe the solution you'd like**
Since we can't do anything in the popup menu for these `AggregateResult` links, the result should not be links at all, and have no functionality when clicked.

**Comment by thephani** [comment]
@tprouvot  can I work on this ?

**Comment by tprouvot** [comment]
Hi @thephani 
I added the help wanted label for this issue so that community members could take it, but to be honest this is not really a priority.

Once you know AgregateResult does not offer any possibility, you don't click it anymore. Or maybe I'm missing something?

**Comment by thephani** [comment]
@tprouvot Right, the result is basically the same - we can hide the link or just disable it.

This is my first time contributing to open source, so I’m looking for something simple to get started.

**Comment by tprouvot** [comment]
Hi @thephani 
Sure, you can start with this one 👍
